roles/openshift_manage_node/tasks/set_default_node_role.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- when: osm_default_node_selector is not defined
```

Comment:
Need to scrape this value from first master's config file, can't trust if it's undefined here because users may have altered their cluster outside of openshift-ansible

Reply:
We're doing something similar for master's config file, I need to find it.

Reply:
roles/etcd/tasks/migration/add_ttls.yml has an example on how to slurp from master.

I would slurp and set_fact on the first_master, then use hostvars to refer to the fact you set.

Reply:
I wish we didn't have to do this, but I agree it's critical that we do. Thanks for mentioning this and pointing at a decent reference.

Reply:
since we have the openshift_master to delegate to here anyway, is there any reason not to just do that logic here?

Reply:
We need this information for determining whether or not to do it on the masters.